### PR TITLE
Updated alerts

### PR DESCRIPTION
**Pull request by ramirodcv**
## Description

Adding more descriptive alerts in test mode.

## TESTING

- [X] Describe what testing have you done for this feature
Tested by hand

## Questions

- [X] Question 1
No points of confusion

## Status

- [X] I have read the guidelines in [CONTRIBUTING.md](https://github.com/itsdawei/dynamically_programmed/blob/master/CONTRIBUTING.md)
- [X] I have formatted my code using `yapf`
- [X] I have linted my code with `pylint`
- [X] I have tested my code by running `pytest`
- [x] I have added a one-line description of my change to the changelog in `HISTORY.md`
- [X] This PR is ready to go

### CODE REVIEW

File: dp/_visualizer.py
Comment (aprasad36) [inline dp/_visualizer.py:1]: This file looks good!

File: demos/edit_distance_1d.py
Comment (aprasad36) [inline demos/edit_distance_1d.py:1]: You should merge edit_distance_1d and edit_distance_2d from main when we have it. The version in this branch doesn't include the documentation, so I think we should remove it from this PR. We should keep the topics of different PRs separate so that it's easier to check previous PRs and isolate issues.

**Review by itsdawei — APPROVED** [review]
Made some formatting changes. There might be a slight revamp of the testing module soon, where we make it more object-oriented by introducing test objects.

File: demos/edit_distance_1d.py
Comment (ramirodcv) [reply demos/edit_distance_1d.py:1]: fixed
